adding information about the branches in NorESMhub/CTSM repository (#14)
NorESMhub/CTSM: README.md

# Branches in NorESMhub/CTSM

> This repository has the following CTSM branches working with NorESM2:

Reviewer: ```suggestion
This repository has the following CTSM branches intended to facilitate working with NorESM2:
```

- *master* - keep track with the master branch of [ESCOMP/CTSM](https://github.com/ESCOMP/CTSM).
  - This branch is used by [NorESMhub/NorESM_LandSites_Platform](https://github.com/NorESMhub/NorESM_LandSites_Platform)
  - This branch can be used for running the most recent version of [FATES](https://github.com/NGEET/fates)

> - *release-clm5.0.14-Nor* - The working branch used by NorESM2. Currently, all the pull requests should be sent to this branch. 

Reviewer 2: I would suggest specifying that only PRs that are not answer changing should be sent to this branch. New developments should be sent elsewhere. Currently there is a clm-nor-dev, which should be used for ongoing developments, but I would also suggest renaming this.

> - **release-clm5.0.14-Nor** - The working branch used by NorESM2. Currently, all the pull requests should be sent to this branch. 

Reviewer: ```suggestion
- **release-clm5.0.14-Nor** - The working branch used by NorESM2. Non-answer changing (with respect to NorESM2s CMIP6 version) pull requests should be sent to this branch. For other changes, new branches discuss with CTSM and NorESM core teams.
```

>   - Tags being developed from this branch are:

Reviewer: ```suggestion
  - Tags in this branch help identifying the link to the NorESM2 releases:
```

>     - release-clm5.0.14-Nor_v1.0.2 - the tag working with NorESM2 master branch and tags: release-noresm2.0.4, release-noresm2.0.3

Reviewer: ```suggestion
    - release-clm5.0.14-Nor_v1.0.2 - the tag working with NorESM2 noresm2 tags: release-noresm2.0.4, release-noresm2.0.3
```

    - release-clm5.0.14-Nor_v1.0.1 - the tag working with NorESM2 tags: release-noresm2.0.2 
    - release-clm5.0.14-Nor_v1.0.0 - the tag working with NorESM2 tags: release-noresm2.0.1, release-noresm2.0.0 

> - **release-cesm2.2.01-Nor** - The branch used for the next phase of NorESM2 (under development).

Reviewer: ```suggestion
- **release-cesm2.2** - Copy of ESCOMP branch, base for release-cesm2.2.01-Nor branch.
- **release-cesm2.2.01-Nor** - The branch used for the next phase of NorESM2 (under development).
```
